pkg/cluster/cluster.go

```diff
@@ -487,7 +491,12 @@ func CreateCluster(cmv1Client *cmv1.Client, config Spec, dryRun bool) (*cmv1.Clu
 }
 
 func isGCPNetworkExists(existingVPC ExistingVPC) bool {
-	return existingVPC.VPCName != "" || existingVPC.ControlPlaneSubnet != "" || existingVPC.ComputeSubnet != ""
+	return existingVPC.VPCName != "" || existingVPC.ControlPlaneSubnet != "" ||
```

Comment:
Optional:  Is checking for existence one of every vpc field correct here ?

Looks like the expectation is that input validation is done by the time this is called.
I think it should suffice if one of the fields i.e  VPCName  exists  or Check  every vpc related field is non-empty before setting it on the clusterBuilder

Reply:
After looking through the code some more , I think the check should simply be   exisitingVPC.enabled

Reply:
`exisitingVPC.enabled` is a derived value that may not be set. I see no reason to change the existing function implementation.